src/SourceBuild/content/repo-projects/roslyn.proj

```diff
@@ -17,13 +17,16 @@
     <BuildArgs Condition="'$(BuildOS)' == 'windows'">$(BuildArgs) $(FlagParameterPrefix)officialSkipApplyOptimizationData true</BuildArgs>
     <BuildArgs Condition="'$(BuildOS)' == 'windows'">$(BuildArgs) $(FlagParameterPrefix)officialSourceBranchName placeholder</BuildArgs>
     <BuildArgs Condition="'$(BuildOS)' == 'windows'">$(BuildArgs) $(FlagParameterPrefix)officialVisualStudioDropAccessToken placeholder</BuildArgs>
+    <BuildArgs>$(BuildArgs) /p:TargetRid=$(TargetRid)</BuildArgs>
```

Comment:
@mmitche this change seems to be causing the PR failure in the source-build leg.

Reply:
I wonder if we only ever do a neutral publish in SB mode. Let me look at the artifacts.

Reply:
Ahh this is a restore error related to filtering. The project doesn't build.